pyocd/rtos/freertos.py

```diff
@@ -505,4 +509,25 @@ def get_is_running(self):
             return False
         return self._target_context.read32(self._symbols['xSchedulerRunning']) != 0
 
-
+    def _get_elf_symbol_size(self, name, addr, defValue):
+        if self._target.elf is not None:
+            symInfo = None
+            try:
+                symInfo = self._target.elf.symbol_decoder.get_symbol_for_name(name)
+            except:
```

Comment:
Can this except clause be made more specific? I try not to have catchall exception handlers in pyocd (which is admittedly sometimes difficult).

Reply:
It should ideally never fail, I did not know what to make it specific to. I looked at other places where this was done. I will change to `RuntimeError` The only reason I added that try/except was because I was dereferencing a bit much and wanted a catch-all.